cmd/config/get/get.go

```diff
@@ -37,12 +38,18 @@ func NewConfigGetCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "get [flags] VARIABLE",
 		Short: "Prints the value of a config variable",
-		Long:  "Prints the value of a config variable. See 'rosa config --help' for supported config variables.",
-		Args:  cobra.ExactArgs(1),
-		Run:   run,
+		Long: fmt.Sprintf("Prints the value of a config variable. Supported variables are: %s.",
+			strings.Join(GetAllConfigProperties(), ", ")),
+		Args: cobra.ExactArgs(1),
+		Run:  run,
 	}
 }
 
+func GetAllConfigProperties() []string {
```

Comment:
@tirthct Shall we move this into `config.go` so it's closer to the `ConfigPropertiesNamesAndDocs()` function and the pattern remains that the `command` implementations simply call a function and do minimal logic themselves?

Reply:
Makes sense. Moved